src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -148,6 +153,16 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
                     "Invalid address",
                     "There was an error processing your address. Please review and try again."
                   )
+                } else if (
+                  errorCode &&
```

Comment:
I see that this is the same as the other `if` statements, but is this line necessary? Checking `if errorCode && errorCode === "some string"` seems extraneous? I could be missing something.

Reply:
I think you're right! Updating...